### PR TITLE
Update privacy policy based on pathogens portal

### PR DESCRIPTION
**Pull request by RMCrean**
Would you be able to take a quick look and see if you're happy with what I've written. I think it looks okay. 

I've essentially reused the pathogens portal policy: https://www.pathogens.se/privacy/ with some minor changes (e.g. we only have 1 contact form).

### CODE REVIEW

File: hugo/content/privacy/_index.md [hunk 1]
```diff
@@ -2,14 +2,11 @@
...
 
-1. Does this also need to be provided in Swedish?
-2. Someone other than Rory needs to check the content/wording etc...
+This page is used to inform website visitors regarding our personal data processing policy. If you choose to use our **Service**, then your personal data will be processed in accordance with this policy.
 
-### Privacy Policy Overview
-
-This site is operated by SciLifeLab Data Centre and this page is used to inform website visitors regarding our data processing and privacy policy. If you choose to use our **Service**, then your personal data will be processed in accordance with this policy.
+The personal information that we collect is used for providing and improving the **Service**. We will not use or share your information with anyone except as described in this policy. All collected personal information will be processed for research purposes, i.e. using the lawful basis of public interest and in accordance with Regulation (EU) 2016/679 of the European Parliament and of the Council of 27 April 2016, the General Data Protection Regulation.
```
Comment (brinkdp) [inline hugo/content/privacy/_index.md:9]: The text on L9 is repeated L19-20. Perhaps this is a typo? A quick look at pathogens.se shows that they only have this text on their L9-equivalent spot.

File: hugo/content/privacy/_index.md [hunk 2]
```diff
@@ -22,6 +19,12 @@ We want to inform you that whenever you visit our **Service**, we collect the in
...
 
+### Contact Form
+
+Our **Service** contains a <a href="/contact" target="_blank">contact form</a> that visitors can use to contact us or send us suggestions. The website visitors may choose to provide their personal information such as their name and e-mail address through this form.
+
+The following parties will have access to processing the personal data provided through the form; SciLifeLab Data Centre, Uppsala University, Kungliga Tekniska högskolan (KTH) and Naturhistoriska riksmuseet (NRM). Your personal data will be deleted when no longer needed, or when stipulated by the archival rules for the university as a government authority. If you want to update or remove your personal data, please contact the controller SciLifeLab Data Centre at Uppsala University using [dsn-eb@scilifelab.se](mailto:dsn-eb@scilifelab.se).
```
Comment (brinkdp) [inline hugo/content/privacy/_index.md:26]: Should we perhaps use the English names for all institutions and not just for UU? My understanding is that their official English versions are: `KTH Royal Institute of Technology, and Swedish Museum of Natural History`. 

(I suspect that there might be inconsistencies about this in other pages in the portal too :smile: )

**Review by brinkdp — APPROVED** [review]

File: hugo/content/privacy/_index.md (hunk 1 above)
Comment (RMCrean) [reply hugo/content/privacy/_index.md:9]: Thanks for spotting this, I've deleted the repeat in commit https://github.com/ScilifelabDataCentre/genome-portal/pull/57/commits/6df7a70e09aa2bde6268ef1d1c71d38993838d3f

File: hugo/content/privacy/_index.md (hunk 2 above)
Comment (RMCrean) [reply hugo/content/privacy/_index.md]: Fair point, I've now made them all use their English names in commit https://github.com/ScilifelabDataCentre/genome-portal/pull/57/commits/6df7a70e09aa2bde6268ef1d1c71d38993838d3f

**Comment by RMCrean** [comment]
I somehow managed to add a file from the playwright tests here... which I've now removed so I can merge this now. Probably easiest to show it Angela in person or when visible on the dev site itself.

**Pull request merged**